be/src/tools/meta_tool.cpp

```diff
@@ -1108,6 +1113,21 @@ int meta_tool_main(int argc, char** argv) {
             return -1;
         }
         std::cout << json << '\n';
+    } else if (FLAGS_operation == "lake_datafile_gc") {
```

Comment:
may consider creating a separate cmdline tool for data file gc, it is not a meta operation any more.